modules/core/src/test/scala/org/scalasteward/core/forge/gitlab/GitLabApiAlgTest.scala

```diff
@@ -232,7 +268,7 @@ class GitLabApiAlgTest extends CatsEffectSuite with Http4sDsl[MockEff] {
   }
 
   test("createPullRequest -- auto merge") {
-    val prOut = gitlabApiAlgNoFork
+    val prOut = gitlabApiAlgAutoMerge
```

Comment:
This test case was identical to "createPullRequest -- no fork". I think the intention was to test payload with `mergeWhenPipelineSucceeds = true` so created `gitlabApiAlgAutoMerge` and used in here.